### PR TITLE
Fix issue 249

### PR DESCRIPTION
**Pull request by Felalolf**
Adds toString methods to (parse tree) types and symbol table entries.

### CODE REVIEW

File: src/main/scala/viper/gobra/frontend/info/base/Type.scala [hunk 1]
```diff
@@ -84,34 +92,38 @@ object Type {
...
-  case class InternalTupleT(ts: Vector[Type]) extends Type
+  case class InternalTupleT(ts: Vector[Type]) extends PrettyType(s"(${ts.mkString(",")})")
 
   case class InternalSingleMulti(sin: Type, mul: InternalTupleT) extends Type
 
-  case class ImportT(decl: PImport) extends Type
+  case class ImportT(decl: PImport) extends PrettyType(decl.formatted)
 
-  case object SortT extends Type
+  case object SortT extends PrettyType("sort")
```
Comment (jcp19) [inline src/main/scala/viper/gobra/frontend/info/base/Type.scala:108]: I believe that `"Type"` is preferable to `"sort"` in order to be consistent with the Go docs (e.g. https://golang.org/src/builtin/builtin.go?s=6524:6544#L160)

**Review by jcp19 — APPROVED** [review]

File: src/main/scala/viper/gobra/ast/frontend/Ast.scala [hunk 2]
```diff
@@ -26,13 +26,15 @@ sealed trait PNode extends Product {
   def pretty(prettyPrinter: PrettyPrinter = PNode.defaultPrettyPrinter): String = prettyPrinter.format(this)
 
   lazy val formatted: String = pretty()
+  lazy val formattedShort: String = pretty(PNode.shortPrettyPrinter)
```
Comment (jcp19) [inline src/main/scala/viper/gobra/ast/frontend/Ast.scala:29]: Do we need a new prettyPrinter? wouldn't the "normal" pretty printer work given that we are only interested in printing the types correctly?

Comment (Felalolf) [reply src/main/scala/viper/gobra/ast/frontend/Ast.scala:29]: I use the short printer for the symbol table entries

File: src/main/scala/viper/gobra/frontend/info/base/Type.scala (hunk 1 above)
Comment (Felalolf) [reply src/main/scala/viper/gobra/frontend/info/base/Type.scala]: While I think that "Type" is a bad name, I agree that picking the same name as Go is preferrable.

**Review by ArquintL — APPROVED** [review]

**Pull request merged**